hazelcast/src/main/java/com/hazelcast/cache/impl/eviction/CacheClearExpiredRecordsTask.java

```diff
@@ -80,7 +79,7 @@
  * </p>
  */
 public class CacheClearExpiredRecordsTask
-        extends ClearExpiredRecordsTask<CachePartitionSegment, ICacheRecordStore> implements OperationResponseHandler {
+        extends ClearExpiredRecordsTask<CachePartitionSegment, ICacheRecordStore> {
```

Comment:
Why is this no longer an `OperationResponseHandler`?

Reply:
Incrementally send them to backups instead of sending upon response. This simplifies the logic.

Please see this line: https://github.com/hazelcast/hazelcast/pull/13997/files#diff-054409d86fe0682574188985d078eaa2R152